src/agent/src/agent_main.cpp

```diff
@@ -0,0 +1,10 @@
+#include "agent.hpp"
+
+#include <iostream>
+
+int main()
+{
+    std::cout << "Wazuh-Agent started" << std::endl;
```

Comment:
This should be replaced by a logger.

Reply:
Done.
I removed it for the time being. Let´s add the logger down the road.